modal_proto/api.proto

```diff
@@ -589,6 +589,7 @@ message CheckpointInfo {
   string runtime_fingerprint = 4;
   int64 size = 5;
   bool checksum_is_file_index = 6;
+  string task_id = 7;
```

Comment:
Can we call this `original_task_id` or something else that makes it clear which task it refers to?

Reply:
ya